### ISSUE TITLE
mails empty

### ISSUE THREAD

**Issue by qyh214** [Sarjuuk/aowow#340]
**Describe the bug and how to reproduce it**
dbc_mailtemplate have data.
but aowow_mails is empty
no other error in cli(cli show success )
in aowow_errors only:
Undefined array key 0   includes\utilities.php  line 163 version 33 phperror 2
**Screenshots**
n/a

**System:**
 - OS: Win10
 - PHP version:8.0.1
 - revision used:33
 - Browser (in case of JavaScript / display errors):n/a
 - AzerothCore: no

To facilitate reproduction and testing, relevant database files are provided

https://mega.nz/folder/lpZzWCgZ#7XBwKp9xOXeFnb7FRQoBvQ

**Comment by jackpoz** [comment]
```
SqlGen::generate() - filling aowow_mails with data
Exception - Column count doesn't match value count at row 1
   /trinity/aowow/includes/libs/DbSimple/Mysqli.php(163)
#0 /trinity/aowow/includes/libs/DbSimple/Mysqli.php(163): mysqli_query()
#1 /trinity/aowow/includes/libs/DbSimple/Database.php(526): DbSimple_Mysqli->_performQuery()
#2 /trinity/aowow/includes/libs/DbSimple/Database.php(221): DbSimple_Database->_query()
#3 /trinity/aowow/setup/tools/sqlgen/mailtemplate.func.php(22): DbSimple_Database->query()
#4 /trinity/aowow/setup/tools/sqlGen.class.php(239): SetupScript@anonymous->generate()
#5 /trinity/aowow/setup/tools/clisetup/sql.func.php(35): SqlGen::generate()
#6 /trinity/aowow/setup/setup.php(40): sql()
#7 /trinity/aowow/aowow(10): require('...')
```

```
DB::Aowow()->query('INSERT INTO ?_mails SELECT *, 0 FROM dbc_mailtemplate');
```

![image](https://user-images.githubusercontent.com/1153754/167265233-3a5732b0-4f2f-444b-8a6d-81997db699ff.png)

![image](https://user-images.githubusercontent.com/1153754/167265245-7043fb18-3ebe-43b3-870d-dea8ae52a126.png)

Notice how `cuFlags` column doesn't exist in dbc_mailtemplate;

Caused by https://github.com/Sarjuuk/aowow/commit/b3e8f5e50f236a65e3a033273199ce881c397659

**Comment by jackpoz** [comment]
@Sarjuuk pinging you for the investigation above

**Comment by Sarjuuk** [comment]
probably fixed in df3694b5394735a18b53e532323dcb9fa5fdf005
